engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -169,11 +169,8 @@ public WorldRendererImpl(Context context, GLBufferPool bufferPool) {
     // TODO: one day the main light (sun/moon) should be just another light in the scene.
     private void initMainDirectionalLight() {
         mainDirectionalLight.lightType = LightComponent.LightType.DIRECTIONAL;
-        mainDirectionalLight.lightColorAmbient = new Vector3f(1.0f, 1.0f, 1.0f);
-        mainDirectionalLight.lightColorDiffuse = new Vector3f(1.0f, 1.0f, 1.0f);
-        mainDirectionalLight.lightAmbientIntensity = 1.0f;
-        mainDirectionalLight.lightDiffuseIntensity = 2.0f;
-        mainDirectionalLight.lightSpecularIntensity = 0.0f;
+        mainDirectionalLight.lightAmbientIntensity = 0.75f;
+        mainDirectionalLight.lightDiffuseIntensity = 0.75f;
```

Comment:
I believe these values were originally bumped up in order that the bug in the shader made sunlight a normal level with clouds.  This is the before and after without the clouds.

Before:
![terasology-160406170158-1280x720](https://cloud.githubusercontent.com/assets/5922109/14394388/e81f38be-fd80-11e5-93f7-d8af17700b45.jpg)

After:
![terasology-160408115448-1280x720](https://cloud.githubusercontent.com/assets/5922109/14394395/ee59e990-fd80-11e5-9721-5986003766d1.jpg)